packages/contracts/contracts/voting/addresslist/AddresslistVotingSetup.sol

```diff
@@ -84,7 +77,7 @@ contract AddresslistVotingSetup is PluginSetup {
             plugin,
             _dao,
             NO_ORACLE,
-            addresslistVotingBase.CHANGE_VOTE_SETTINGS_PERMISSION_ID()
+            addresslistVotingBase.UPDATE_PLUGIN_SETTINGS_PERMISSION_ID()
```

Comment:
Probably not for this PR, but on the SDK we rather name the operation like `setPluginSettings`.

Reply:
See https://github.com/aragon/core/pull/161#discussion_r1049792084